modules/rostests/apitests/ntdll/NtAllocateVirtualMemory.c

```diff
@@ -198,7 +200,10 @@ CheckMemory2(
 }
 
 VOID
-CheckSize(ULONG_PTR Base, SIZE_T InSize, SIZE_T ExpectedSize)
+CheckSize(
```

Comment:
This (and the VOID below) is just noise, please don't do this.

Reply:
Dropped.